models/AccessToken.js

```diff
@@ -0,0 +1,28 @@
+var mongoose = require("mongoose");
+var { conn } = require("../config/db");
+
+var access_token = mongoose.Schema(
+    {
+      user_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "user",
+        unique: true,
+      },
+      access_token: {
+        type: String,
+      },
+      expiry: {
+        type: Date,
+        expires: "60s",
```

Comment:
expire 60 sec ?

Reply:
added for token expire in 60s will change it to 10min